cypress/integration/Homepage.test.js

```diff
@@ -12,38 +12,49 @@ describe("Landing Page", () => {
         expect(el.find("a")).to.have.class("govuk-link");
       });
     cy.get(".submasthead__content").should("exist");
-    cy.get(
-      ".govuk-grid-column-two-thirds > :nth-child(1) > .govuk-link"
+    cy.get(".govuk-accordion__open-all").should("exist");
+    cy.get(".govuk-accordion__open-all").click();
+    cy.get(".govuk-grid-column-two-thirds > :nth-child(1)").should("exist");
+    cy.contains("School curriculum and teaching").should("exist");
+    cy.contains("Pupil wellbeing and safety").should("exist");
+    cy.contains(
+      "Closures, exams and managing a school or early years setting"
     ).should("exist");
+    cy.contains("Funding and support for education providers").should("exist");
+    cy.contains("Student accommodation, travel and financial support").should(
+      "exist"
+    );
+    cy.contains("Further and higher education and vocational training").should(
+      "exist"
+    );
   });
 
-  it('Checks the "Teaching during coronavirus" hyperlink works', () => {
+  it('Checks the "Remote education during coronavirus" hyperlink works', () => {
     cy.get(".govuk-grid-column-two-thirds .govuk-link")
-      .eq(0)
+      .eq(3)
       .should((el) => {
-        expect(el).to.contain.text(
-          "Remote education during coronavirus (COVID-19)"
-        );
+        expect(el).to.contain.text("Remote education during coronavirus");
         expect(el).to.have.attr("href");
         expect(el.attr("href")).to.equal("/teachers/");
       })
-      .click();
+      .click({ force: true });
```

Comment:
why forced?

Reply:
Element are in invisible state.

Reply:
would be ideal to avoid force clicking, as it is not visible it's not a true test of full functionality.